### DISCUSSION TITLE
Создание плейлиста на основе рандомных треков существующего

### DISCUSSION THREAD

**Discussion by zhegal** [Chimildic/YaMuTools#33] [Идея] [answered]
Здравствуйте. Первый день пользуюсь расширением, очень доволен.

**Ситуация**: у меня есть плейлист, допустим "мне нравится", в котором больше тысячи треков.
Хотелось бы иметь возможность выдернуть из этой тысячи рандомные треки (по указанному количеству в настройках) и на их основе создать новый плейлист.

**Comment by kisoid** [comment]
Так есть уже такая функциональность...

**Reply by zhegal** [reply]
Нет. Сейчас есть возможность создать "похожий плейлист". Я же имею ввиду что у меня есть плейлист из 1000 и мне нужно создать новый, где из этой тысячи будет выбрано какое-то количество в случайном порядке. Не похожие, а именно из этого же плейлиста.

**Reply by kisoid** [reply]
Можно создать новый плейлист, в "Мне нравится" нажать "копировать треки в другой плейлист", а в новом нажать "Сортировка/Случайно" - получится то же самое... Прослушаешь из нового плейлиста столько, сколько захочешь и удалишь.

**Reply by kisoid** [reply]
И что мешает просто запустить случайное проигрывание в исходном плейлисте и выключить, когда надоест?

**Comment by sergvpr** [comment]
Я думаю, это полезная функция - отфильтровать количество треков в плейлисте.
Например, чтобы скачать плейлист и послушать в пути. Качать все 1000 много места займет, а прослушать не удастся.

**Comment by Chimildic** [comment, marked as answer]
Добавил в следующее обновление, ожидайте
